postgres.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ankane/pgvector
+
+RUN apt-get update && apt-get install -y --no-install-recommends postgresql-15-cron
+
+COPY ./db-init.sh /docker-entrypoint-initdb.d/db-init.sh
```

Comment:
What about executing the .sh commands in here, instead of having a diff file?

Reply:
We need to run these commands after Postgres is initialised and not at build time.

> The PostgreSQL data directory (/var/lib/postgresql/data) is usually not created until the PostgreSQL server is initialized, which typically happens when the container is run, not when it's being built.

The files in the `docker-entrypoint-initb.d/` folder  will run directly when Postgres is initialised

Reply:
I can look into a work around so we have every thing in one file

Reply:
Found a solution